app/pods/agendas/route.js

```diff
@@ -66,6 +66,10 @@ export default class AgendasRoute extends Route {
   @action
   loading(transition) {
     // see snippet in https://api.emberjs.com/ember/3.27/classes/Route/events/loading?anchor=loading
+    if (transition.from === null || transition.from.name !== this.routeName) {
+      return true;
+    }
```

Comment:
Aanpassing is correct, maar kan je misschien dezelfde logica gebruiken als hier voor consistency? https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/6f45f6a47f61df7ccdb3943bf4408ec3c02955eb/app/pods/publications/overview/_base/route.js#L74-L90